src/app/backend/handler/apihandler.go

```diff
@@ -1035,6 +1046,25 @@ func (apiHandler *APIHandler) handleGetReplicationControllerList(request *restfu
 	response.WriteHeaderAndEntity(http.StatusOK, result)
 }
 
+func (apiHandler *APIHandler) handleOverview(request *restful.Request, response *restful.Response) {
```

Comment:
It was removed. Why do you add this? You might solve merge conflicts wrongly, otherwise please explain.